integration/spark-datasource/src/test/scala/org/apache/spark/sql/carbondata/datasource/TestCreateTableUsingSparkCarbonFileFormat.scala

```diff
@@ -438,9 +438,11 @@ class TestCreateTableUsingSparkCarbonFileFormat extends FunSuite with BeforeAndA
   private def clearDataMapCache(): Unit = {
     if (!spark.sparkContext.version.startsWith("2.1")) {
       val mapSize = DataMapStoreManager.getInstance().getAllDataMaps.size()
-      DataMapStoreManager.getInstance()
-        .clearDataMaps(AbsoluteTableIdentifier.from(writerPath))
-      assert(mapSize > DataMapStoreManager.getInstance().getAllDataMaps.size())
+      if (mapSize > 0) {
```

Comment:
Why need this check?
I think there may be some hidden problems in other tests that lead us uncertain about whether this value is 0 or not. As a result, current modification will hide those problems, which means this modification is **not proper** in my opinion.

Reply:
Fixed it.